src/app/shared/profile-modal/profile-modal.component.css

```diff
@@ -10,3 +10,27 @@
 .modal-backdrop {
   background-color: rgb(20, 20, 20);
 }
+
+.modal-body {
+  max-height: calc(100vh - 130px);
```

Comment:
how does this look without the 130px? can we avoid using specific pixel values?

Reply:
We looked over this and got it looking great for both desktop and mobile